### DISCUSSION TITLE
Upgrading PayloadCMS from 1.x to 2.x: Is It Necessary to Modify the Website When CMS and Website Are Separate?

### DISCUSSION THREAD

**Discussion by kouohhashi** [payloadcms/payload#4921] [Q&A] [answered]
I would like to upgrade PayloadCMS from version 1.x to 2.x. 

If the CMS and website are separate, is it unnecessary to make changes on the website side?

**Comment by kouohhashi** [comment, marked as answer]
It seems frontend works without any change.

**Reply by schebitz** [reply]
How was your experience in the backend after updating to 2.0? Would you recommend it? Did you try to upgrade to 3.0 already?

**Reply by kouohhashi** [reply]
To tell the truth, I sticked to 1.0 because 1.0 was good enough. but I'm thinking to setup 3.0 and import data later.
